### ISSUE TITLE
Broken use of uninit memory: test_target_teams_distribute_reduction_bitor.F90

### ISSUE THREAD

**Issue by tob2** [SOLLVE/sollve_vv#128]
`tests/4.5/target_teams_distribute/test_target_teams_distribute_reduction_bitand.F90` and
`tests/4.5/target_teams_distribute/test_target_teams_distribute_reduction_bitor.F90` have:

```
    REAL(8),DIMENSION(N, 32):: randoms
    INTEGER:: result, host_result, x, y, z, errors, itr_count
...
             IF (randoms(x, y) .gt. true_margin) THEN
```
Here, `y` is an array index but uninitialized.

Likewise for for OpenMP `/5.0/`

**Comment by spophale** [comment]
Thanks @tob2 !

**Comment by jhdavis8** [comment]
See PR #134

**Comment by jhdavis8** [comment]
Fix has been merged (PR #134)
